hack/make/.integration-test-helpers

```diff
@@ -7,6 +7,25 @@ bundle_test_integration_cli() {
 	go_test_dir integration-cli $DOCKER_INTEGRATION_TESTS_VERIFIED
 }
 
+bundle_test_integration() {
+	TESTFLAGS="$TESTFLAGS -v -test.timeout=60m"
```

Comment:
@mlaventure hum right :angel: I just copied the others `bundle_xx` but yep, make sens